### PR TITLE
Refine big merge

### PR DESCRIPTION
**Pull request by lightwalker-eth**

### CODE REVIEW

File: apps/examples.nameguard.io/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nextjs-sdk",
+  "name": "nameguard-examples",
```
Comment (notrab) [inline apps/examples.nameguard.io/package.json:2]: This is an example, we can get rid of this and version.

Comment (lightwalker-eth) [reply apps/examples.nameguard.io/package.json:2]: This is useful during the build process.

CleanShot 2024-05-16 at 16 12 57

Comment (lightwalker-eth) [reply apps/examples.nameguard.io/package.json:2]: Removed version :+1:

File: apps/examples.nameguard.io/postcss.config.cjs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   plugins: {
     tailwindcss: {},
-    autoprefixer: {},
-  },
-}
+    autoprefixer: {}
+  }
+}
```
Comment (FrancoAguzzi) [inline apps/examples.nameguard.io/postcss.config.cjs:6]: Blank line needed

**Review by FrancoAguzzi — APPROVED** [review]

File: apps/examples.nameguard.io/postcss.config.cjs (hunk 2 above)
Comment (lightwalker-eth) [reply apps/examples.nameguard.io/postcss.config.cjs:6]: Interesting. What creates this need? I added one as requested but don't see what it changes. If some different fix is needed please feel welcome to action that directly in this PR :+1:

**Pull request merged**